### ISSUE TITLE
Provide pynari linux arm64 build (+ Macos and Win ARM64 builds too)?

### ISSUE THREAD

**Issue by oscarbg** [ingowald/pynari#5]
Hi,
really just read about pynari while on a Mac.. so running on a linux arm64 VM:

pip install pynari
ERROR: Could not find a version that satisfies the requirement pynari (from versions: none)
ERROR: No matching distribution found for pynari

I'm on Ubuntu 24.10 so python 3.12 and the closest it's a x64 wheel:
https://files.pythonhosted.org/packages/db/40/008489c572846a324952e0801c426903e13c917e2172804618c74aa641df/pynari-1.0.36-cp312-cp312-manylinux_2_34_x86_64.whl

**Comment by ingowald** [comment]
I currently only gave Barney hooked up as backend, and Barney requires optix/CUDA, so building for Mac probably doesn't really make sense :-/

**Comment by ingowald** [comment]
agreed, let's keep this open. i might also try to create a cpu-only version for non-cuda machines ... might not generate the same images, but better than nothing.

**Comment by ingowald** [comment]
I finally managed to build some cpu-only version, and made that work also on Mac. So "in theory" there should now be MacBook version available via pip as well. let me know if/how it works, happy to fix things as required (I bought myself a MacBook to make that happen, but dont' use it daily, so no guarantees)

**Comment by ingowald** [comment]
latest pynari builds shuold all have mac and windows binaries, and cpu support too

**Comment by ingowald** [comment]
Happy to hear! There's still a few things missing on mac (denoising in particular, and speed isn't exactly great yet), but it's working now, and that's a huge thing. If you ever get anything you want to share: we'd be happy to hear about it!
